### PR TITLE
Added Storybook

### PR DESCRIPTION
**Pull request by JamieB-gu**
## Why?

Introduces Storybook for developing components in a browser. I've added a couple of stories for the `Img` component; the image used is taken from [this article](https://www.theguardian.com/travel/2020/jan/27/rail-sail-how-to-book-perfect-holiday-without-flying).

The Webpack-related Storybook config changes make use of [`ts-loader`](https://github.com/TypeStrong/ts-loader) to handle TypeScript, as the Babel preset only [supports a subset](https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats) of features.

This PR is based against the `img-component` branch while #1 is open.

**Note:** I had some trouble getting the Babel preset working for the Emotion `css` prop, so I've switched to using the [JSX pragma](https://emotion.sh/docs/css-prop#jsx-pragma) method for expediency - it works out-of-the-box with a minimal Webpack config. That said, it's probably worth a second attempt at this in future.

## Changes

- Added storybook and peer dependencies
- Added script to launch storybook
- Added config for storybook including webpack tweaks for TypeScript
- Added stories for `Img` component
- Used JSX pragma for Emotion

### CODE REVIEW

File: src/components/img.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            alt: some('Demo image'),
+            width: 5644,
+            height: 3387,
+            caption: none,
+            credit: none,
+            rawCaptionHtml: none,
+            role: Role.Standard,
+        }}
+        sizes="80vw"
+        className={some(css`width: 40vw; height: 30vw; display: block;`)}
```
Comment (mxdvl) [inline src/components/img.stories.tsx:54]: If you're forcing your width to `40vw`, then your sizes should be `40vw`. Sizes is used to force an [intrinsic size][] on the element and relates to the viewport.

[intrinsic size]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes

Comment (JamieB-gu) [reply src/components/img.stories.tsx:54]: Good spot, fixed.

File: src/components/img.stories.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ const Placeholder: FC = () =>
             rawCaptionHtml: none,
             role: Role.Standard,
         }}
-        sizes="80vw"
+        sizes="40vw"
         className={some(css`width: 40vw; height: 30vw; display: block;`)}
```
Comment (mxdvl) [inline src/components/img.stories.tsx]: I think we can go further and extract this value. If your units are consistent you can do something like this:

```suggestion
        sizes="${displayWidth}"
        className={some(css`width: ${displayWidth}; height: ${displayWidth * ratio}; display: block;`)}
```

(Do not commit this change as I don't know whether you need to divide or multiply by `ratio`)

Comment (JamieB-gu) [reply src/components/img.stories.tsx]: I've extracted some of the props out and made this change using those values.

**Pull request merged**